sumologic/resource_sumologic_monitors_library_monitor_test.go

```diff
@@ -139,9 +143,12 @@ func TestAccSumologicMonitorsLibraryMonitor_schemaTriggerConditionValidations(t
 		testName := "terraform_test_invalid_monitor_" + testNameSuffix
 
 		resource.Test(t, resource.TestCase{
-			PreCheck:     func() { testAccPreCheck(t) },
-			Providers:    testAccProviders,
-			CheckDestroy: testAccCheckMonitorsLibraryMonitorDestroy(monitorsLibraryMonitor),
+			PreCheck:  func() { testAccPreCheck(t) },
+			Providers: testAccProviders,
+			CheckDestroy: func() resource.TestCheckFunc {
```

Comment:
for my learning, i understand what you made a change now but how was old code working?

Reply:
`testAccCheckMonitorsLibraryMonitorDestroy` did take 1 param, however it was unused. I removed it as part of this PR.